src/nomad_material_processing/vapor_deposition/cvd/movpe.py

```diff
@@ -0,0 +1,215 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import (
+    TYPE_CHECKING,
+)
+
+from nomad.datamodel.data import EntryData
+from nomad.datamodel.metainfo.annotations import (
+    ELNAnnotation,
+    ELNComponentEnum,
+    SectionProperties,
+)
+from nomad.datamodel.metainfo.plot import PlotSection
+from nomad.datamodel.metainfo.workflow import (
+    Link,
+)
+from nomad.metainfo import (
+    Quantity,
+    SchemaPackage,
+    Section,
+    SubSection,
+)
+
+from nomad_material_processing.vapor_deposition.cvd.general import (
+    CVDSource,
+    CVDStep,
+    Rotation,
+)
+from nomad_material_processing.vapor_deposition.general import (
+    ChamberEnvironment,
+    Pressure,
+    SampleParameters,
+    SubstrateHeater,
+    Temperature,
+    VaporDeposition,
+    VolumetricFlowRate,
+)
+
+if TYPE_CHECKING:
+    pass
+
+from nomad.config import config
+
+m_package = SchemaPackage()
+
+configuration = config.get_plugin_entry_point(
+    'nomad_material_processing.vapor_deposition.cvd:movpe_schema',
+)
+
+
+class FilamentTemperature(Temperature):
+    """
+    Temperature of a heated element used to keep the substrate hot.
+    """
+
+    value = Quantity(
+        type=float,
+        unit='kelvin',
+        a_eln=ELNAnnotation(
+            defaultDisplayUnit='celsius',
+        ),
+        shape=['*'],
+    )
+    set_value = Quantity(
+        type=float,
+        description='The set value(s) (i.e. the intended values) set.',
+        shape=['*'],
+        unit='kelvin',
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity,
+            defaultDisplayUnit='celsius',
+            label='Set value',
+        ),
+    )
+
+
+class MovpeSampleParameters(SampleParameters):
+    filament_temperature = SubSection(
+        section_def=FilamentTemperature,
+    )
+
+
+class MovpeChamberEnvironment(ChamberEnvironment):
+    uniform_gas_flow_rate = SubSection(
+        section_def=VolumetricFlowRate,
+    )
+    pressure = SubSection(
+        section_def=Pressure,
+    )
+    throttle_valve = SubSection(
+        section_def=Pressure,
+    )
+    rotation = SubSection(
+        section_def=Rotation,
+    )
+    heater = SubSection(
+        section_def=SubstrateHeater,
+    )
+
+
+class StepMovpe(CVDStep, PlotSection):
+    """
+    Growth step for MOVPE
+    """
+
+    m_def = Section(
+        a_eln=None,
+    )
+    sample_parameters = SubSection(
+        section_def=MovpeSampleParameters,
+        repeats=True,
+    )
+    sources = SubSection(
+        section_def=CVDSource,
+        repeats=True,
+    )
+    environment = SubSection(
+        section_def=MovpeChamberEnvironment,
+    )
+
+
+class Movpe(VaporDeposition, EntryData):
```

Comment:
I'm not sure what guideline we use for PascalCase and acronyms at the moment. We have `CVDStep` but `MovpeEntryPoint`. According to some Microsoft guidelines it should be `CvdStep` since the acronym is longer than 2 letters. But I guess most important is to be consistent. Let's discuss on Discord.

Also, shouldn't this be:
```suggestion
class Movpe(ChemicalVaporDeposition):
```
Not sure we want `EntryData` on this level.

Reply:
BTW, this might seem like nitpicking but it will break future imports so we should decide on something and stick to it I guess.